plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogAccessOperations.java

```diff
@@ -365,37 +367,37 @@ public void testSelectSystemTable()
             // select from $history
             assertGlueMetastoreApiInvocations("SELECT * FROM \"test_select_snapshots$history\"",
                     ImmutableMultiset.builder()
-                            .addCopies(GET_TABLE, 1)
+                            .add(GET_TABLE)
                             .build());
 
             // select from $snapshots
             assertGlueMetastoreApiInvocations("SELECT * FROM \"test_select_snapshots$snapshots\"",
                     ImmutableMultiset.builder()
-                            .addCopies(GET_TABLE, 1)
+                            .add(GET_TABLE)
                             .build());
 
             // select from $manifests
             assertGlueMetastoreApiInvocations("SELECT * FROM \"test_select_snapshots$manifests\"",
                     ImmutableMultiset.builder()
-                            .addCopies(GET_TABLE, 1)
+                            .add(GET_TABLE)
                             .build());
 
             // select from $partitions
             assertGlueMetastoreApiInvocations("SELECT * FROM \"test_select_snapshots$partitions\"",
                     ImmutableMultiset.builder()
-                            .addCopies(GET_TABLE, 1)
+                            .add(GET_TABLE)
```

Comment:
This & others

Unnecessary change.

(i don't want to argue whether `.add(x)` vs `.add(x, 1)` should be used, but let's treat it as an unrelated change)